### ISSUE TITLE
Relax dependency constraints

### ISSUE THREAD

**Issue by Jell** [backup/backup#456]
It would be great if you could relax the constraints on the runtime dependencies, they all are specified at the patch level which makes backup impossible to peacefully cohabit with other gems.

It is said in your wiki that backup is not supposed to be used as a dependency, but I would very much like to use it as a dependency in my rails app. The reason is simple: I want to be able to deploy a backup task for every app I deploy.

So basically replacing all `=` with `~>`  and removing the patch level from the specs, that would be great.

Doable?
